src/AdoPat.Test/PatManagerTest.cs

```diff
@@ -30,6 +32,11 @@ public class PatManagerTest
         // This is a dummy authorization ID, not valid in any real contexts.
         private readonly Guid authorizationId = new Guid("ee0c5586-a96f-4a44-b1d9-8613028b1078");
 
+        private readonly string cacheKey = string.Join('-', OrganizationHash, DisplayNameHash, ScopeHash);
+
+        private Extensions.Logging.ILogger logger;

```

Comment:
Can you not just say `ILogger logger` and put the rest in usings?

Reply:
Uhh... maybe? I just copied what was done in another similar test class.

https://github.com/AzureAD/microsoft-authentication-cli/blob/cc9319143a6b15d8523616d9d3f0f478ff18150e/src/MSALWrapper.Test/LockedTest.cs#L23